gameday/transport.go

```diff
@@ -28,6 +31,98 @@ func AddRoutes(router *mux.Router, svc *Service, logger logrus.FieldLogger) {
 	router.HandleFunc("/api/v1/gamedays/cancel/lookup", handleLookupGamedays(svc, logger))
 }
 
+func HandleConfigure(router *mux.Router, logger logrus.FieldLogger) http.HandlerFunc {
```

Comment:
could we try to run a unit test here and catch cases when migrations are failing or when we configure DB badly?